README.md

```diff
@@ -134,3 +134,105 @@ Tests
 =====
 
 See [kafka/README](kafka/README)
+
+
+
+Getting started
+===============
+
+Installing librdkafka
+---------------------
+
+    git clone https://github.com/edenhill/librdkafka.git
+    cd librdkafka
+    ./configure --prefix /usr
```

Comment:
Until it is merged should we specify the `events_api` branch before doing any compiling here?